docs/tutorials/generator/generator_only.md

```diff
@@ -0,0 +1,19 @@
+# Using only the generator without any retrieved content.
+
+If you want to use the generator model directly without retrieving any context from the processed data, you can do so by setting the `top_k` parameter to 0 in the retriever module. 
+
+### Using config file
+
+```python
+retriever:
+  retriever_top_k: 20
```

Comment:
If we set retriever_top_k in the config to 0 without setting agrag.retriever_module.top_k = 0, will it work?

Reply:
Yeah you can do either one.